conf/crg.config

```diff
@@ -5,11 +5,30 @@ params {
     config_profile_url = 'http://www.linux.crg.es/index.php/Main_Page'
 }
 
-process {
-    executor = 'crg'
-    queue    = 'short-centos79,long-centos79'
+def host = System.getenv("HOSTNAME")
+
+if (host ==~ /^genoa64.*/) {
+    process {
+        executor = 'slurm'
+        queue    = 'genoa64'
+        clusterOptions = {
+            task.time <= 1.h    ? '--qos=vshort' :
```

Comment:
This is a bit hard to read for me, one might want to consider a switch with a case ranges (see e.g. https://stackoverflow.com/a/66986654) .